onnxruntime/core/optimizer/transpose_optimizer/optimizer_api_impl.cc

```diff
@@ -879,6 +879,10 @@ const std::unordered_set<std::string_view>& GetORTLayoutSensitiveOps() {
       // onnx_layout_transformation::HandleResize is used.
       ,
       "Resize"
+#endif
+#if defined(USE_QNN)
+      ,
+      "InstanceNormalization"
```

Comment:
InstanceNormalization is a general layout sensitive op for all EPs, should add it to layout_sensitive_ops
https://github.com/microsoft/onnxruntime/blob/6c8538f0862f213ba8df3ffde67e81024a7a810f/onnxruntime/core/optimizer/transpose_optimizer/transpose_optimizer.cc#L2042-L2046

Reply:
Made the change